typescript/sdk/src/ism/types.ts

```diff
@@ -374,3 +374,10 @@ export const IsmConfigSchema = z.union([
   OffchainLookupIsmConfigSchema,
   InterchainAccountRouterIsmSchema,
 ]);
+
+export enum RadixIsmTypes {
+  MERKLE_ROOT_MULTISIG = 'MerkleRootMultisigIsm',
+  MESSAGE_ID_MULTISIG = 'MessageIdMultisigIsm',
+  ROUTING_ISM = 'RoutingIsm',
+  NOOP_ISM = 'NoopIsm',
+}
```

Comment:
I see that the `radix-sdk` package defines a similar type to this one with the difference that that one is a union type could we maybe have this enum replace that type and be exported from the `radix-sdk` package instead? so we can avoid this "duplication"

https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/radix-sdk/src/utils/types.ts#L29-L38